protocol/src/main/kotlin/models/Message.kt

```diff
@@ -40,19 +42,20 @@ sealed class Message {
   init {
     // json schema validate
     validate()
-    if (signature != null) {
```

Comment:
is this block not needed anymore?

Reply:
sig check is only needed when you're parsing an existing message, and `init` doesn't actually get called when you use the jackson object mapper